examples/coap/project-conf.h

```diff
@@ -40,7 +40,7 @@
 #define PROJECT_ERBIUM_CONF_H_
 
 /* Custom channel and PAN ID configuration for your project. */
-/* #define RF_CHANNEL                    26 */
+/* #define IEEE802154_CONF_DEFAULT_CHANNEL                    26 */
 /* #define IEEE802154_CONF_PANID     0xABCD */
```

Comment:
I wonder if we should we get rid of those

Reply:
agree, will remove